janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/StoreFeatures.java

```diff
@@ -39,6 +39,12 @@
      */
     boolean hasUnorderedScan();
 
+    /**
+     * Whether this storage backend supports keys order while running parallel scans
```

Comment:
"Whether this storage backend supports keys order while running parallel scans" seems to suggest when `hasInconsistentScan()` is true, key order is consistent while running parallel scans.

My suggestion is, we could make this method "hasConsistentScan()". Then the doc could be,

> Whether this storage backend supports a consistent key order among different scans. If it supports ordered scan, it must support consistent key scan. If it doesn't support ordered scan, it may or may not support consistent key scan. If the consistent scan is not supported, the backend shall support {@link KeyColumnValueStore#getKeys(MultiSlicesQuery, StoreTransaction)}

Reply:
Sounds perfect!

Reply:
fixed